pyrsa/util/file_io.py

```diff
@@ -115,3 +115,17 @@ def read_dict_pkl(file):
         file = open(file, 'rb')
     data = pickle.load(file)
     return data
+
+
+def remove_file(file):
+    """ Deletes file from OS if it exists
+
+    Args:
+        file (str, Path):
+            a filename or opened readable file
```

Comment:
I am not sure this will work with an opened readable file as this is neither a str or Path instance. Specifically, remove_file(open('myrdm.pkl', 'rb')) would not work as far as I can see. I am not sure whether this is what is meant by "a filename or opened readable file".

Reply:
Hi Ben, I think it was meant to work on an opened file as well. I now added lines to make this happen.